source/views/index.html.haml

```diff
@@ -1,6 +1,17 @@
 ---
 title: GradientDaze
 ---
+.explainer.animated.zoomIn
+  %header
+    =inline_svg("G-Logotype.svg", class: "g-logotype")
 
-%h1.teaser
-  Coming Winter 2016
+  %h1.teaser
+    Gesture based gradients never felt so good
+
+  .built-by
+    built by
+    %a{:href => "//www.thoughtbot.com"} thoughtbot, inc
+
+.mobile-device.animated.zoomIn
```

Comment:
You are using camelCase for `.zoomIn`, but you don't use it anywhere else, so I would just do `.zoom-in`